client/aws/aws.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/support/supportiface"
 	"github.com/giantswarm/microerror"
 
-	"github.com/giantswarm/aws-operator/service/controller/v2/resource/cloudformation/adapter"
+	"github.com/giantswarm/aws-operator/service/controller/v18/adapter"
```

Comment:
This smells and should be refactored. We only import this for account ID validation, which is pretty basic. We do the validation in a couple of places though. Any idea how this should look like?

Reply:
@giantswarm/sig-operator WDYT?